content/app/design/insight/_index.en.md

```diff
@@ -1,33 +1,26 @@
-komme vekk fra møtebordet og bruke post-its på veggene.
-- **Hvem er brukerne og hvilke behov har de?** Vi anbefaler at dere starter med å definere målgruppe og 
-brukerbehov så tidlig som mulig i prosjektet. Lag en oversikt over hvem brukerne er. I de fleste tilfellene 
-skal man lage løsninger som fungerer for flere brukergrupper. Er det innbyggere eller aktører i næringslivet? 
-Hvilken bakgrunn har de? Er de digitale eller ikke så vant til å bruke digitale tjenester? 
-I denne fasen gjøres det typisk innledende innsiktsarbeid i form av intervjuer, observasjoner eller fokusgruppe.
-- **Kontaktpunkter for brukeren** Sett opp en oversikt/brukerreise over brukerens kontaktpunkt. 
-Eksempel på hvordan en fragmentert brukerreise kan fortone seg: 
-![Kontaktpunkter](kontaktpunkter.png "Kontaktpunkter") Skal Altinn være skjult for 
-sluttbruker eller er det Altinns grensesnitt som skal benyttes? Stemmer dette med hvordan brukeren forventer 
-å få løst sin oppgave? Er det mulig å tilby brukerne den relevante delen av altinn i den konteksten de er i, 
-i stedet for å sende dem frem og tilbake?
-- **Hva er databehovet?** Behovet for en digital løsning starter gjerne med at det er behov for å samle inn 
-data. Start med å undersøke om andre har dataene du trenger og om disse er tilgjengelige for gjenbruk. Det 
-kan du sjekke i <a href="https://data.norge.no/">Felles datakatalog</a>. Hvis ikke - hvordan kan du spørre 
-på en måte som krever minst mulig av brukerne? Noen ganger kan det være en idé å prøve noe annet enn opplagte 
-løsninger hvor man spør i form av et skjema.
+- **What are the current challenges?** Start by defining the problem. What are you going to solve? 
+It is easier to create a solution with a clearly defined scope for your problem. It might be a good 
+idea to gather your project group, and to look for a nice wall to put post-it notes on.
+- **Who are the users and what are their needs?** 
+We recommend starting by defining your target audience and their requirements as early as possible during the project. 
+Create an overview of your users are. In most cases your service should be usable by multiple usergroups. 
+Are they citizens or do they represent a company? What background do they have? 
+Are they digitaly inclined or not used to digital services? 
+This phase normally starts with interviews, observations or a focus group.
+- **Contact points for the user** Create an overview of the points of contact for the user. What does the customer see during their use of your service? 
+One way to do this is pictured below:
+![Kontaktpunkter](kontaktpunkter.png "Kontaktpunkter") Should Altinn be hidden from the end user or will you be using Altinns internal UI? Does this match how the user expects to complete their task? Is it possible to offer the customer the relevant part of Altinn in the context they are in, instead of sending them back and fourth? 
```

Comment:
```suggestion
![Points of contact](kontaktpunkter.png "Points of contact") Should Altinn be hidden from the end user or will you be using Altinns internal UI? Does this match how the user expects to complete their task? Is it possible to offer the customer the relevant part of Altinn in the context they are in, instead of sending them back and forth? 
```